packages/@aws-cdk/core/test/custom-resource-provider/test.custom-resource-provider.ts

```diff
@@ -20,6 +20,16 @@ export = {
     // THEN
     test.ok(fs.existsSync(path.join(TEST_HANDLER, '__entrypoint__.js')), 'expecting entrypoint to be copied to the handler directory');
     const cfn = toCloudFormation(stack);
+
```

Comment:
Is this still needed after normalization?

Reply:
This is in `core`, while normalization is in `assert`. Dependencies don't work out.